src/styles/main.scss

```diff
@@ -0,0 +1,79 @@
+$stopwatch-size: 500px;
+$hand-width: 10px;
+$small-hand-height: $stopwatch-size / 3;
+$center: $stopwatch-size / 2;
+
+//a minute is arranged for 6s since it is boring to watch
+$animation-minute-time: 6s;
+
+@mixin watchSize {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  border-radius: 50%;
```

Comment:
this does not seem reusable which is the idea behind mixins. also if your mixin does not take in any arguments, you should use a placeholder instead